docs/software/container-development-guide.md

```diff
@@ -0,0 +1,74 @@
+Container Development Guide
+===========================
+
+Creating a New Container Image
+------------------------------
+
+1. Create a Git repository whose name is prefixed with `docker-`, e.g. `docker-frontier-squid`
+1. Create a `README.md` file describing the software provided by the image
+1. Create a `LICENSE` file containing the [Apache 2.0 license text](https://www.apache.org/licenses/LICENSE-2.0.txt)
+1. Create a `Dockerfile` based off of the OSG Software Base image:
+
+        FROM opensciencegrid/software-base:fresh
+
+        LABEL maintainer OSG Software <help@opensciencegrid.org>
+        
+        RUN yum clean all && \
+            yum update -y 
```

Comment:
```suggestion
            yum update -y && yum clean all && rm -rf /var/cache/yum/*
```
on the other hand, you do need to clean up afterward.

Reply:
You should combine the yum update and the subsequent yum install into one RUN command.